### ISSUE TITLE
Higher than normal latency after migration of feature toggles

### ISSUE THREAD

**Issue by laineymajor** [department-of-veterans-affairs/va.gov-team#56481]

**Comment by rmtolmach** [comment]
- Trying increasing minReplicas in https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/1735

**Comment by rmtolmach** [comment]
The following PRs and commits were added to the vets-api `parent-helm-charts` branch:
- https://github.com/department-of-veterans-affairs/vets-api/pull/12358
- https://github.com/department-of-veterans-affairs/vets-api/pull/12359
- https://github.com/department-of-veterans-affairs/vets-api/commit/6db8fab676205aa079b2c079702361c0d624b0be
  - https://github.com/department-of-veterans-affairs/vets-api/commit/141aebd0f3bd4a64aa30a43b1b9c516450146503
  - https://github.com/department-of-veterans-affairs/vets-api/commit/c5f5cc9e237d2e494aabd51ea1e01b6b45242889
- https://github.com/department-of-veterans-affairs/vets-api/commit/86752c8a2ae3f8afdeacec5e58cc31e0e69ab461
- https://github.com/department-of-veterans-affairs/vets-api/commit/c54a54ae8fb1f9353a7d068cc9ed56b7533a0a82

@oseasmoran73 and @RachalCassity ran load testing on Staging
```
$ cd ../devops
$ docker run --rm -v `pwd`/loadtest:/loadtest -i locustio/locust:2.14.2 \
  -u 250 -r 5 -t 5m --headless --only-summary -H https://staging-api.va.gov/ \
  -f /loadtest/search/search_locust.py
```

From Slack. @LindseySaari says
> Theory: Rachal and I think that our thread/pool metric is being divided by the number of pods. Docs prove what we are reading is true, so that's why were not scaling.
**Example**: our capactiy/thread calculation = 100 (so this should scale right) BUT this is being DIVIDED by number of pods.
100/4 pods = 25.
We also tested at 2 pods 100/2 pods = 50
TLDR; We are trying `targetValue` instead of `targetAverageValue`
From the docs: 
> > If we use targetValue, it's values is the absolute value. For targetAverageValue we are defining the individual value for each replica (so it divides the value by the number of replicas)

Now we are scaling on the correct metric.

**Comment by laineymajor** [comment]
Good to close
